src/components/SameLanguageSubtitle.js

```diff
@@ -423,9 +423,9 @@ export default function SameLanguageSubtitles({
         (subtitles) => {
             const suburl = vtt2url(subtitles);
             url2sub(suburl).then((urlsub) => {
-                // setSubtitle(formatSub(urlsub));
+                setSubtitle(formatSub(urlsub));
                 setSubtitleEnglish(formatSub(urlsub));
-                //localStorage.setItem('subtitle', JSON.stringify(urlsub));
+                localStorage.setItem('subtitle', JSON.stringify(urlsub));
                 localStorage.setItem('subtitleEnglish', JSON.stringify(urlsub));
```

Comment:
Earlier, I think this was commented-out on purpose. Because when you switch to translation mode, it would display same subtitles on both the sides (if we are to do like this).

Can you please check once with Alpana or Harsh (whoever made the change earlier)